Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+let package = Package(
+    name: "NexaSwift",
+    platforms: [
+        .macOS(.v15),
+        .iOS(.v18),
```

Comment:
is it possible to lower the iOS and macOS version requirements?
llama.cpp writes iOS as v14
https://github.com/ggerganov/llama.cpp/blob/master/Package.swift#L51-L53

Reply:
Our package uses some functions (like `init(validating:as:)`) that are only available in newer OS versions, which sets our current minimum requirements. We can consider backward compatibility in future versions based on community needs.